h/search/index.py

```diff
@@ -26,7 +27,17 @@ class Window(namedtuple('Window', ['start', 'end'])):
     pass
 
 
-def index(es, annotation, request, target_index=None):
+def index(models_annotation, request):
```

Comment:
No `es` (client) param needed here. No `index` param needed here. (We may have to add back an optional `index` param because I think some of the re-indexing code uses `index_old`'s `target_index` param to index to the new index. Let's just call it `index` not `target_index` though.)